diff --git a/esm/external/monaco-lsp-client/out/index.js b/esm/external/monaco-lsp-client/out/index.js
index e3ace9c821b14855e093f40e17fa7a63d330e000..187ea430fecee900e6aedbe24bc6b198320c2e79 100644
--- a/esm/external/monaco-lsp-client/out/index.js
+++ b/esm/external/monaco-lsp-client/out/index.js
@@ -2620,7 +2620,6 @@ var LspDiagnosticsFeature = class extends Disposable {
         relatedDocumentSupport: true
       }
     } }));
-    debugger;
     this._register(this._connection.connection.registerNotificationHandler(api.client.textDocumentPublishDiagnostics, (params) => this._handlePublishDiagnostics(params)));
     this._register(this._connection.capabilities.registerCapabilityHandler(capabilities.textDocumentDiagnostic, true, (capability) => {
       const disposables = new DisposableStore();
@@ -2951,7 +2950,7 @@ var TextDocumentSynchronizer = class extends Disposable {
   }
   _getOrCreateManagedModel(m) {
     if (!this._started) throw new Error("Not started");
-    const uriStr = m.uri.toString(true).toLowerCase();
+    const uriStr = m.uri.toString(true);
     let mm = this._managedModels.get(m);
     if (!mm) {
       mm = new ManagedModel(m, this._server);
@@ -2966,7 +2965,7 @@ var TextDocumentSynchronizer = class extends Disposable {
     return mm;
   }
   translateBack(textDocument, position) {
-    const uri = textDocument.uri.toLowerCase();
+    const uri = textDocument.uri;
     const textModel = this._managedModelsReverse.get(uri);
     if (!textModel) throw new Error(`No text model for uri ${uri}`);
     return {
@@ -2975,7 +2974,7 @@ var TextDocumentSynchronizer = class extends Disposable {
     };
   }
   translateBackRange(textDocument, range) {
-    const uri = textDocument.uri.toLowerCase();
+    const uri = textDocument.uri;
     const textModel = this._managedModelsReverse.get(uri);
     if (!textModel) throw new Error(`No text model for uri ${uri}`);
     return {
@@ -3010,7 +3009,7 @@ var ManagedModel = class extends Disposable {
     super();
     this._textModel = _textModel;
     this._api = _api;
-    const uri = _textModel.uri.toString(true).toLowerCase();
+    const uri = _textModel.uri.toString(true);
     this._api.textDocumentDidOpen({ textDocument: {
       languageId: _textModel.getLanguageId(),
       uri,
